src/index.html

```diff
@@ -11,6 +11,41 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+      class="search-form search-form_big"
+      data-qa="big"
+    >
+      <label
+        for="big"
+        class="search-form_label search-form_label_big"
+      >
+        <input
+          type="text"
+          data-qa="keypress"
+          id="big"
+          class="search-form_input search-form_input_big"
+          placeholder="Try ''Los Angeles''"
```

Comment:
```suggestion
          placeholder="Try ''Los Angeles''"
```
Потрібно використовувати подвійні лапки, а не дві одинарних. Тому і тести зараз не проходять

https://user-images.githubusercontent.com/66570142/142000928-63edc497-d266-4185-aee6-2ede07d15e29.mov